repos/system_upgrade/el7toel8/actors/kernel/checkkernelmodules/files/removed_modules.txt

```diff
@@ -0,0 +1,254 @@
+3w-9xxx
```

Comment:
I'd like to see a comment line describing where did we get the list. We got it by comparing RHEL 7 and RHEL 8 modules. But what version/compose of these systems?